tests/unit/amo/pages/TestCollectionList.js

```diff
@@ -227,7 +227,10 @@ describe(__filename, () => {
     expect(userCollections).toHaveLength(2);
 
     userCollections.forEach((collection, index) => {
-      const expected = createInternalCollection({ detail: collections[index] });
+      const expected = createInternalCollection({
+        detail: collections[index],
+        numberOfAddons: collections[index].addon_count,
```

Comment:
If `detail` is `collections[index]`, why do we need to explicitly pass `numberOfAddons`? Can't this be retrieved inside `createInternalCollection()`?

Reply:
Good point. I don't need to be passing `numberOfAddons` in a couple of places where I'm doing it. `createInternalCollection` can figure it out from `detail`.